sonata-project/admin-bundle/3.30/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Sonata\\CoreBundle\\SonataCoreBundle": ["all"],
+        "Knp\\Bundle\\MenuBundle\\KnpMenuBundle": ["all"],
+        "Sonata\\AdminBundle\\SonataAdminBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
```

Comment:
why do we need this? and why do we need the `.gitignore` file?

Reply:
Did you mean to create a `.gitkeep` file?

Reply:
Yes `.gitkeep` should be used.

Reply:
Sorry, I was wrong. We should use `.gitignore`. See https://github.com/symfony/recipes/pull/334

Reply:
Thanks for digging that up! I'll probably stick with `.gitkeep` in my own projects when I really need to version control a directory, because I'd rather use a convention for what it was meant than a misuse a standard in a confusing way.